way_to_home/way_to_home/settings.py

```diff
@@ -82,7 +84,7 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'ENGINE': 'django.db.backends.postgresql',
```

Comment:
```suggestion
        'ENGINE': 'django.db.backends.postgresql',
```
Return `postgresql_psycopg2`

Reply:
The `django.db.backends.postgresql_psycopg2` backend has been renamed to `django.db.backends.postgresql` in Django 1.9.

Reply:
done